templates/v2/user/settings.html.twig

```diff
@@ -1,73 +1,41 @@
-                    {{ include( app.user.isBeneficiaire
-                        ? 'v2/user/user_beneficiary_settings_form.html.twig'
-                        : 'v2/user/user_settings_form.html.twig'
-                        , {'settingsForm': userForm}) }}
-                </div>
-                <div class="text-primary pb-2 row mt-4">
-                    <div class="col pt-2">
-                        <i class="fas fa-lock font-size-1-3"></i>
-                        <span class="font-size-1-3 bold uppercase">
-				        &nbsp; {{ 'user.parametres.changerMotDePasse'|trans }}
-				    </span>
-                    </div>
-                </div>
-                <hr class="bg-primary my-0">
-                <br/>
-                <div class="container w-75 justify-content-center mb-4">
-                    {{ include('v2/user/change_password_form.html.twig', {'askCurrentPassword': true, 'passwordForm': passwordForm}) }}
-                </div>
-                {% if app.user.isBeneficiaire %}
-                    <div class="text-center mb-5">
-                        <a href="{{ path('user_delete') }}"
-                           class="font-size-1 vault-green">{{ 'delete_your_account'|trans }}</a>
-                    </div>
-                {% endif %}
-            {% endblock %}
-        </div>
-    </div>
-{% endblock body %}
+{% block title %}
+    {{ 'user_settings_title_1'|trans }}
+    <span class="{{ getUserTextColor() }}">
+        {{ 'user_settings_title_2'|trans }}
+    </span>
+{% endblock %}
 
-{% block styles %}
-    {{ parent() }}
-    {{ encore_entry_link_tags('appV2style') }}
+{% block subTitle %}
+    {{ 'user_parameters_subtitle'|trans({'%subject%' : app.user.prenom ~ ' ' ~ app.user.nom})|upper }}
 {% endblock %}
 
-{% block javascripts %}
-    {{ parent() }}
-    {{ encore_entry_script_tags('appV2') }}
+{% block content %}
+    <div class="container w-75 justify-content-center">
+        {{ include( app.user.isBeneficiaire
+            ? 'v2/user/_beneficiary_settings_form.html.twig'
+            : 'v2/user/_pro_settings_form.html.twig',
+            {form: userForm},
+        ) }}
+    </div>
+    <div class="text-primary pb-2 row mt-4">
+        <div class="col pt-2 font-size-1-3">
+            <i class="fas fa-lock me-1"></i>
+            <span class="bold uppercase">
+                {{ 'user.parametres.changerMotDePasse'|trans|upper }}
+            </span>
+        </div>
+    </div>
+    <hr class="bg-primary my-0 hr-2">
+    <div class="container w-75 justify-content-center my-4">
+        {{ include('v2/user/change_password_form.html.twig', {'askCurrentPassword': true, 'passwordForm': passwordForm}) }}
+    </div>
+    {% if app.user.isBeneficiaire %}
+        <div class="text-center mb-5">
+            <a href="{{ path('user_delete') }}"
+               class="font-size-1 text-green">{{ 'delete_your_account'|trans }}</a>
+        </div>
+    {% endif %}
 {% endblock %}
```

Comment:
I think we could do it differently by having two child templates, one for each kind of users. In the parent template you can have an empty ` {% settingsBlock %} ` that will be different in each child, and add the delete button on the beneficiaire child template. In the controller you will render the proper template depending on the user kind.